spec/features/scientists/show_spec.rb

```diff
@@ -0,0 +1,89 @@
+    @se_4 = ScientistExperiment.create!(scientist_id: @scientist5.id, experiment_id: @experiment1.id)
+    @se_5 = ScientistExperiment.create!(scientist_id: @scientist5.id, experiment_id: @experiment3.id)
+    @se_6 = ScientistExperiment.create!(scientist_id: @scientist5.id, experiment_id: @experiment4.id)
+  end
+
+  context "As a visitor" do
+    describe "when I visit a scientist's show page," do
+      it "I see all of that scientist's information (name, specialty, alma mater)," do
+        visit scientist_path(@scientist1)
+
+        within("#scientist_info")
+
+        expect(page).to have_content("Name: #{@scientist1.name}")
+        expect(page).to have_content("Specialty: #{@scientist1.specialty}")
+        expect(page).to have_content("Alma Mater: #{@scientist1.university}")
+
+        expect(page).to_not have_content("Name: #{@scientist5.name}")
+        expect(page).to_not have_content("specialty: #{@scientist5.specialty}")
+        expect(page).to_not have_content("Alma Mater: #{@scientist5.university}")
+      end
+
+      it "I see the name of the lab where this scientist works," do
+        visit scientist_path(@scientist1)
+
+        within("#scientist_info")
+
+        expect(page).to have_content("Lab: #{@scientist1.lab.name}")
+        expect(page).to_not have_content("Lab: #{@scientist5.lab.name}")
+      end
+
+      it "and I see the names of all of the experiments this scientist is running." do
+        visit scientist_path(@scientist1)
+
+        within("#scientist_experiments")
+
+        expect(page).to have_content("#{@experiment1.name}")
+        expect(page).to have_content("#{@experiment2.name}")
+        expect(page).to have_content("#{@experiment4.name}")
+
+        expect(page).to_not have_content("#{@experiment3.name}")
+
+      end
+
+      it "Then next to each experiment's name, I see a button to remove that experiment from that scientist's work load" do
+        visit scientist_path(@scientist1)
+        
+        expect(page).to have_button("Remove")
+      end
+
+      it "When I click that button for one experiment, I'm brought back to the scientist's show page, and I no longer see that experiment's name listed" do
+        visit scientist_path(@scientist1)
+        within("#scientist_experiment")
+        expect(page).to have_content("#{@experiment1.name}")
+
+        within("experiment-#{@experiment1.id}")
+        click_button "Remove #{@experiment1.name}"
+        
+        expect(page).to_not have_content("#{@experiment1.name}")
+      end
+
+      it "And when I visit a different scientist's show page that is working on that same experiment, then I see that the experiment is still on the other scientist's work load" do
+        visit scientist_path(@scientist5)
+        within("#scientist_experiment")
+        expect(page).to have_content("#{@experiment1.name}")
+      end      
+    end
+  end
+end
```

Comment:
I don't know if your `within` blocks are actually working properly without their respective do and end pieces. This could also be a new syntax that I am not familiar with.